plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1061,7 +1073,8 @@ def main():
         mutually_exclusive=[('delete', 'revert'), ('delete', 'update'), ('revert', 'update'), ('clone', 'update'), ('clone', 'delete'), ('clone', 'revert')],
         required_together=[('api_token_id', 'api_token_secret')],
         required_one_of=[('name', 'vmid'), ('api_password', 'api_token_id')],
-        required_if=[('state', 'present', ['node'])],
+        required_if=[('state', 'present', ['node']),
+                     ('bios', 'ovmf', ['efidisk0'])],
```

Comment:
This is a breaking change. Before when `bios=ovmf`, `efidisk0` was **not** required before. Why is this necessary?

Reply:
As stated in the first comment, this is not necessary, I included it in order to follow the recommendations stated in the Proxmox VE Documentation (https://pve.proxmox.com/pve-docs/pve-admin-guide.html#qm_virtual_machines_settings see 10.2.10). I hadn't considered the fact that it would be considered as a breaking change, and I can easily remove it from this PR.

Reply:
Please remove it in this case. We do not accept breaking changes without a longer deprecation preriod upfront.

Reply:
It might make sense to print a warning (or even a deprecation warning) if `bios=ovmf` if `efidisk0` is not provided. That should only be done if `bios=ovmf` without `efidisk0` is not useful in **any** case. If there is at least one useful case, there must not be a warning.

Reply:
Basically, if the VM doesn't require boot order persistence or Secure Boot, then the `efidisk0` parameter isn't needed, even though the `bios=ovmf` option is provided. So if I understand your comment correctly, there shouldn't be a warning if `bios=ovmf` is set without setting `efidisk0`.

Reply:
removed in commit 995d1eb